### PR TITLE
Lending: Banking file upload

### PR DESCRIPTION
**Pull request by pzaichkina**
# Description

Adding a banking file upload function page

## Type of change

- [X] New document(s)/updating existing

### CODE REVIEW

File: docs/lending/functions/file-upload.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+If you already have a banking data provider, you can still benefit from our bank statement categorization functionality by manually uploading that banking data to Codat.
+
+You can do this in two ways:
+
+1. Upload banking data files and download the resulting categorized bank statement [in our Portal](/lending/functions/file-upload#upload-in-portal).
+2. Push the banking data records and get the resulting categorized bank statement [via our API](/lending/functions/file-upload#upload-via-api).
+
+:::caution Prerequisites
+
+To use the data upload functionality, you need to create a Codat [company](../../terms/company) first. We walk you through this in [Get started with Lending API](/lending/get-started#use-lending-api).
```
Comment (dcoplowe) [inline docs/lending/functions/file-upload.md:19]: Love it.

File: docs/lending/functions/file-upload.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+| Transactions | [CSV template](https://static.codat.io/public/templates/lending/transactions-template.csv) | `id`, `accountId`, `currency`, `description`, `amount`, `postedDate`, `code`                                                                                            | See the [Banking: Transaction](/lending-api#/schemas/BankingTransaction) schema |
+
+You can check the progress of the upload by navigating to **Data history > Pull history** of the company. Once the upload is complete, download the [Enhanced cash flow](/lending/features/excel-download-overview#feature-components) report to view the resulting categorized bank statement in **Lending > Reports**. 
+
+To add, amend or remove records, upload a CSV file that contains the updated dataset you want recorded in Codat, and it will replace the existing dataset. For example, if you perform a monthly statement upload, each new CSV file must contain the data for all previously uploaded months as well as the new month.
```
Comment (dcoplowe) [inline docs/lending/functions/file-upload.md:36]: Having to reupload isn't a very good user experience.

File: docs/lending/functions/file-upload.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+To change it, delete the connection, set the configuration for the new `connectionId`, and reupload the data.
+
+:::
+
+2. **Start the upload session**
+
+Use the [Start upload session](/lending-api#/operations/start-bank-statement-upload-session) endpoint to initiate a bank statement upload session for a given company. A session is a one-time process that lets accounts and transactions to be uploaded to Codat. 
+
+You can only have one active session per data type at a time. Additionally, the session will time out if no data is uploaded for 90 minutes. You can complete or cancel a session using the [End upload session](lending-api#/operations/end-bank-statement-upload-session) endpoint.
```
Comment (dcoplowe) [inline docs/lending/functions/file-upload.md:62]:
```suggestion
You can only have one active session per data type at a time. Additionally, the session will time out if no data is uploaded after 90 minutes. You can complete or cancel a session using the [End upload session](lending-api#/operations/end-bank-statement-upload-session) endpoint.
```

**Review by dcoplowe — APPROVED** [review]

File: docs/lending/functions/file-upload.md (hunk 2 above)
Comment (pzaichkina) [reply docs/lending/functions/file-upload.md:36]: feed back to fintech-lending please :P

**Pull request merged**